app/controllers/admin/ticket_scannings_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Admin
+  class TicketScanningsController < Admin::BaseController
+    before_action :authenticate_user!
+    before_action :find_physical_ticket_by_token
```

Comment:
and I am even thinking that this shouldn't happen right? So, this would be really unexpected, so maybe even correct to raise the exception.